src/fabric.c

```diff
@@ -51,9 +51,9 @@
 #endif
 
 struct fi_prov {
-	struct fi_prov		*next;
-	struct fi_provider	*provider;
-	void			*dlhandle;
+	struct fi_prov *next;
+	struct fi_provider *provider;
+	void *dlhandle;
```

Comment:
Minor nit: this kind of whitespace change (which happens in a few places) should be in a separate commit -- not part of the addition of the log stuff.  There's a bunch of whitespace changes in this PR that make finding the real changes a little more difficult.